src/components/Layout/Header.tsx

```diff
@@ -200,35 +227,46 @@ const Header: FunctionComponent<Props> = props => {
                                         data-toggle="collapse"
                                         data-target="#customer-menu"
                                         aria-expanded="false"
-                                        onClick={() => setCustomerMenuOpen(!customerMenuOpen)}
-                                        onKeyDown={() => setCustomerMenuOpen(!customerMenuOpen)}
+                                        onClick={() => setuseCaseMenuOpen(!useCaseMenuOpen)}
+                                        onKeyDown={() => setuseCaseMenuOpen(!useCaseMenuOpen)}
                                         tabIndex={0}
                                     >
-                                        Customers
+                                        Use Cases
                                         <ChevronDownIcon className="icon-inline ml-1" />
                                     </span>
                                     <ul
                                         id="customer-menu"
                                         className={`small-menu collapse navbar-collapse ${
-                                            customerMenuOpen ? 'show' : 'hide'
+                                            useCaseMenuOpen ? 'show' : 'hide'
                                         }`}
                                     >
                                         <li className="nav-link" role="presentation">
-                                            <Link href="/case-studies">Case studies</Link>
+                                            <Link href="/use-cases/">All Use Cases</Link>
```

Comment:
You can omit the trailing slash

```suggestion
                                            <Link href="/use-cases">All Use Cases</Link>
```